### DISCUSSION TITLE
impossibile modificare plancia energia

### DISCUSSION THREAD

**Discussion by cele66** [indomus/forum#1837] [HUB: Home Assistant]
Ho bisogno di aiuto per modificare la Plancia Energia in HA, durante la configurazone ho inserito ei sensori errati e ora non riesco più a modificarla.
c'è possibilità di farlo?

grazie

**Comment by calcioscacchi** [comment]
ciao, da "impostazioni -->plance puoi accedere al menu di modifica della plancia energia semplicemente  cliccandoci sopra.

**Reply by cele66** [reply]
non ho nessun tasto modifica da impostazioni/plance, non trovo nulla che mi permetta la modifica

**Reply by calcioscacchi** [reply]
Non ho parlato di tasto modifica,  clicca sopra il nome della plancia

**Reply by cele66** [reply]
ci clicco ed entro nella sezione dove inserire le entità per il monitoraggio, dove ho inserito quelle errate che vorrei modificare
![image](https://user-images.githubusercontent.com/111089251/185364479-857a87a1-42c9-4eb0-82e0-52955eefaace.png)

**Reply by calcioscacchi** [reply]
Ok,  ora è più chiaro il problema. Strano non te le faccia cambiare,  tocca aspettare qualcuno più esperto di me
